src/scss/components/_card.scss

```
@@ -10,12 +10,18 @@
   padding: $grid-margin-8;
   box-sizing: border-box;
 
+  @include bem.element('header') {
+    &--padded {
+      @include rows(1, padding-bottom);
+    }
```

```suggestion
    @include bem.modifier('padded') {
      @include rows(1, padding-bottom);
    }
```

The `&--` syntax results in a selector `.openforms-card .openforms-card__header--padded`, while the mixin syntax results in the stricter `.openforms-card .openforms-card__header.openforms-card__header--padded`

In BEM, you should *not* be able to use a modifier without also specifying the base that was modified.

Do:

`<span class="block__element block__element--modifier">`

Don't:

`<span class="block__element--modifier">`

Making the selector specific enforces correct classnames in the markup.

---

I've seen this before because of some string utilities being missing :thinking: I'll see if I can look into it tomorrow